projects/plugins/boost/composer.json

```diff
@@ -24,6 +24,7 @@
 		"automattic/jetpack-lazy-images": "@dev",
 		"automattic/jetpack-my-jetpack": "@dev",
 		"automattic/jetpack-plugin-deactivation": "@dev",
+		"automattic/jetpack-plugins-installer": "0.2.x-dev",
```

Comment:
Based on the github actions, sounds like this needs to be `@dev` :)

Reply:
Thanks! Fixed with bb5998613b72ede395457fa30c30a8b46c8f51d9.